cmd/soroban-cli/src/events.rs

```diff
@@ -0,0 +1,304 @@
+use clap::{ArgEnum, Parser};
+use termcolor::{Color, ColorChoice, StandardStream, WriteColor};
+use termcolor_output::colored;
+
+use soroban_env_host::xdr::{self, ReadXdr};
+
+use crate::rpc::{Client, Event};
+use crate::{rpc, snapshot, utils};
+use crate::{HEADING_RPC, HEADING_SANDBOX};
+
+#[derive(Parser, Debug)]
+#[clap()]
+pub struct Cmd {
+    /// The first ledger sequence number in the range to pull events
+    #[clap(short, long)]
+    start_ledger: u32,
+
+    /// The last (and inclusive) ledger sequence number in the range to pull events
+    /// https://developers.stellar.org/docs/encyclopedia/ledger-headers#ledger-sequence
+    #[clap(short, long)]
+    end_ledger: u32,
+
+    /// Formatting options for outputted events
+    #[clap(long, arg_enum, default_value = "pretty")]
+    format: OutputFormat,
+
+    /// RPC server endpoint
+    #[clap(long,
+        env = "SOROBAN_RPC_URL",
+        help_heading = HEADING_RPC,
+        conflicts_with = "events-file",
+        conflicts_with = "follow",
+    )]
+    rpc_url: Option<String>,
+
+    /// Local event store (likely generated by `invoke`) to pull events from
+    #[clap(
+        long,
+        parse(from_os_str),
+        value_name = "PATH",
+        env = "SOROBAN_EVENTS_FILE",
+        help_heading = HEADING_SANDBOX,
+        conflicts_with = "rpc-url",
+    )]
+    events_file: Option<std::path::PathBuf>,
+
+    /// Whether or not we should keep watching the events file and displaying
+    /// them in real time.
+    #[clap(
+        short, long,
+        help_heading = HEADING_SANDBOX,
+        conflicts_with = "rpc-url",
+    )]
+    follow: bool,
```

Comment:
If we're not implementing this now let's remove it. Better to not leave features in a broken state.